### ISSUE TITLE
508/UX - Chat Controls menu not keyboard accessible

### ISSUE THREAD

**Issue by iamjolly** [GSA-TTS/10x-ai-sandbox#503]
## Problem Description

The Chat Controls menu has multiple keyboard accessibility problems. 

1. When the Controls button is activated to show/expand the "drawer" for Chat Controls, the focus does not move to the Chat Controls items when tabbing but instead moves to the Profile button and then to other elements in the `main` content area of the page. This is likely due to the Chat Controls contents being inserted by JavaScript into the DOM near the end of the document instead of directly after the Controls `button` that initiated them to be revealed.
2. The Valves, System Prompt, and Advanced Params accordion controls are not keyboard accessible. They are incorrectly coded as `div` elements that do not natively receive keyboard focus. It is unlikely that there are keyboard event listeners for these as well. They should be `button` elements with `aria-expanded` attributes and corresponding values based on their states.
3. There are no visible focus indicators for any of the items in the chat controls menu due to improper and redundant barrier-creating use of tailwind-css `outline-none` classes on every single focusable item in this menu.

## Screenshot

<img width="831" alt="multiple problems related to keyboard accessibility shown in screenshot of chat controls menu interface" src="https://github.com/user-attachments/assets/c90d19c9-6427-4028-887d-afe4b23b49d7" />
